server/src/main/scala/org/ensime/model/ModelBuilders.scala

```diff
@@ -39,29 +39,33 @@ trait ModelBuilders { self: RichPresentationCompiler =>
     }
   }
 
-  def locateSymbolPos(sym: Symbol, needPos: PosNeeded): Option[SourcePosition] =
-    if (sym == NoSymbol || needPos == PosNeededNo) None
+  def locateSymbolPos(sym: Symbol, needPos: PosNeeded): Option[SourcePosition] = {
+    if (sym == NoSymbol || needPos == PosNeededNo)
+      None
     else if (sym.pos != NoPosition) {
-      if (needPos eq PosNeededYes)
+      if (needPos == PosNeededYes || needPos == PosNeededAvail) {
```

Comment:
Here we have the source position and yet we would only return it if it was 'Yes' rather than 'Yes' and 'if available'